MonoGame.Framework/BoundingFrustum.cs

```diff
@@ -4,103 +4,178 @@
 
 using System;
 using System.Diagnostics;
-using System.Text;
 
 namespace Microsoft.Xna.Framework
 {
+    /// <summary>
+    /// Defines a frustum for intersection operations.
+    /// </summary>
     [DebuggerDisplay("{DebugDisplayString,nq}")]
     public class BoundingFrustum : IEquatable<BoundingFrustum>
     {
         #region Private Fields
 
-        private Matrix matrix;
-        private readonly Vector3[] corners = new Vector3[CornerCount];
-        private readonly Plane[] planes = new Plane[PlaneCount];
+        private Matrix _matrix;
+        private readonly Vector3[] _corners = new Vector3[CornerCount];
+        private readonly Plane[] _planes = new Plane[PlaneCount];

```

Comment:
I know how tempting it is to do these cleanups...  I really hate members without the leading `_` myself.  However you *really* should avoid doing this.  We do not want "cleanup" of code.

Go ahead and leave this in now, but please avoid it in the future.